webpack.dev.js

```diff
@@ -1,29 +1,30 @@
+const { join } = require('path');
 const { smartStrategy } = require('webpack-merge');
 const { HotModuleReplacementPlugin, NamedModulesPlugin, WatchIgnorePlugin } = require('webpack');
 const commonConfig = require('./webpack.common');
-const flowEditorConfig = require('./flowEditor.config.dev');
 
-const devServerPort = 9000;
+const DEV_SERVER_PORT = 9000;
 
 const devConfig = {
     entry: [
         'react-hot-loader/patch',
-        'webpack-dev-server/client?http://localhost:9000',
+        `webpack-dev-server/client?http://localhost:${DEV_SERVER_PORT}`,
         'webpack/hot/only-dev-server'
     ],
     devtool: 'source-map',
     devServer: {
+        contentBase: [join(__dirname, 'src'), join(__dirname, 'assets')],
         compress: true,
         hot: true,
-        port: devServerPort,
+        port: DEV_SERVER_PORT,
         proxy: {
             '/assets/flows': {
                 bypass: req => {
-                    const { query: { uuid } } = req;
+                    const { query: { uuid }, originalUrl } = req;
                     if (uuid) {
-                        return `/test_flows/${uuid}.json`;
+                        return `/flows/${uuid}.json`;
                     }
-                    return req.originalUrl;
+                    return originalUrl.replace('/assets', '');
```

Comment:
have to replace 'assets' in url since `/assets` is included in our dev server's content base